fhir_zulip_nlp/__main__.py

```diff
@@ -1,5 +1,8 @@
 """FHIR Zulip NLP Analysis"""
-from fhir_zulip_nlp.fhir_zulip_nlp import run, CONFIG
+try:
```

Comment:
Tried to add some resiliency to make it run whether doing (1) `python -m fhir_zulip_nlp` or (2) `python fhir_zulip_nlp/fhir_zulip_nlp.py`. Idk if I got it quite right though. I recommend running via (1).